src/components/common/Card/DesignCard.tsx

```diff
@@ -0,0 +1,45 @@
+import { MouseEvent } from 'react';
+type Props = {
+  title?: string;
+  edit?: string;
+  image?: string;
+  state?: 'normal' | 'active';
+  onContextMenu?: (e: MouseEvent<HTMLDivElement>) => void;
+};
+
+export default function DesignCard({
+  title = '디자인 이름이름',
+  edit = '최근에 수정됨',
+  image,
+  state = 'normal',
+  onContextMenu,
+}: Props) {
+  const className = {
+    getState: () => {
+      switch (state) {
+        case 'normal':
+          return ['border-d900 bg-d10 hover:border-d700 hover:bg-d30', 'text-d700 hover:text-d700'];
+        case 'active':
+          return ['border-d900 bg-d900 hover:border-d700 hover:bg-d700', 'text-d10 hover:text-d10'];
+        default:
+          return ['', ''];
+      }
+    },
+  };
+
+  return (
+    <div
+      className={`flex flex-col w-[320px] h-[280px] p-4 gap-4 
+    ${className.getState()[0]} border rounded-lg`}
+      onContextMenu={onContextMenu}
+    >
+      <div className="bg-d10 w-full h-full border flex justify-center items-center">
+        <img src={image} alt="designCardImage" />
+      </div>
+      <div className={`${className.getState()[1]}`}>
```

Comment:
문법 자체는 이렇게 표현해줘도 괜찮겠네요~
다만 스타일을 이렇게 적용하는 건 좋지 않은 것 같습니다.
```suggestion
      <div className={className.getState()[1]}>
```